larsim/LegacyLArG4/OpFastScintillation.cxx

```diff
@@ -2192,15 +2192,15 @@ namespace larg4 {
 
   // ---------------------------------------------------------------------------
 
-  constexpr double
-  acos_table(const double x)
-  {
-    if (x < 0. || x > 1.) {
-      std::cout << "Range out of bounds in acos_table, only defined in [0, 1]" << std::endl;
-      exit(0);
-    }
-    return acos_arr[std::round(acos_bins * x)];
-  }
+  // constexpr double
+  // acos_table(const double x)
+  // {
+  //   if (x < 0. || x > 1.) {
+  //     std::cout << "Range out of bounds in acos_table, only defined in [0, 1]" << std::endl;
+  //     exit(0);
+  //   }
+  //   return acos_arr[std::round(acos_bins * x)];
+  // }
```

Comment:
Recommend removing instead of commenting out.

Reply:
Lines removed with d829239.
It was cleanup but it was also related to this PR. The variable `acos_arr` was unused, causing `sbndcode` to fail compilation. I can of course change to compiler flags in `sbndcode` when it compiles this part, but I think it is cleaner to just remove the unused variable.